src/server/editorServices.ts

```diff
@@ -1197,6 +1202,11 @@ namespace ts.server {
         private forEachConfigFileLocation(info: ScriptInfo,
             action: (configFileName: NormalizedPath, canonicalConfigFilePath: string) => boolean | void,
             projectRootPath?: NormalizedPath) {
+
+            if (this.ignoreConfigFiles) {

```

Comment:
Do you really need this flag.. Even with syntax mode, calculating project's compiler options would be ideal isn't it? (eg. Target etc will have impact in parsing say what is considered identifier etc)

Reply:
I was unaware that the target could affect parsing.  Do you have an example?

Also, we're trying to pull in as few files as possible, so we don't really want to load the rest of the configured project.

Reply:
I had a quick look in the scanner and it looks like ES5 allows some additional Unicode characters in identifiers.  However, my impression is that passing the same text to an ES3 parser would result in an error, rather than a different valid parse tree.  Is that correct?  If so, I think we can safely ignore the difference (possibly by explicitly specifying `ScriptTarget.Latest`).

Reply:
Superficially, it looks like only scanning is affected.  Did I miss something in the parser?